ipfscluster.go

```diff
@@ -83,6 +83,8 @@ type IPFSConnector interface {
 	// RepoStat returns the current repository size and max limit as
 	// provided by "repo stat".
 	RepoStat(context.Context) (*api.IPFSRepoStat, error)
+	// RepoGC returns the status of the garbage collection "repo/gc"
+	RepoGC(context.Context) error
```

Comment:
@lanzafame , Is this correct? Please let me know.

Reply:
Yes, looks correct